interop-testing/src/main/proto/grpc/testing/messages.proto

```diff
@@ -160,3 +162,13 @@ message ReconnectInfo {
   bool passed = 1;
   repeated int32 backoff_ms = 2;
 }
+
+message LoadBalancerStatsRequest {
+  int32 num_rpcs = 1;
+  int32 timeout_sec = 2;
+}
+
+message LoadBalancerStatsResponse {
+  map<string, int32> rpcs_by_peer = 1;
+  int32 num_without_peer = 2;
```

Comment:
What does this field mean?

Reply:
Renamed to `num_failures` (it doesn't exactly correspond with failed RPCs, but for the purpose of the load balancer stats request these are rpcs which failed to produce stats)

Reply:
I don't care too much about choosing the name of the field. As there is always subtleties, it's worth a protodoc to clarify any subtleties that the user should be aware of.

Reply:
Ok, done.